### ISSUE TITLE
any sample available implementing HTTPProxy and its associated dataportal targeting .netcore/.netstandard

### ISSUE THREAD

**Issue by EllinaExt** [MarimerLLC/cslaforum#674]
since, support of WCFProxy and WCFPortal is not anymore supported in csla 4.8.1. is there any sample available implementing HTTPProxy and its associated dataportal targeting .netcore/.netstandard ?

**Version and Platform**
CSLA version: 4.8.100
OS: Windows, Linux, iOS, Android, etc.
Platform: >net Standard, .NetCore.

**Comment by rockfordlhotka** [comment]
Yes, the ProjectTracker sample (`Samples/ProjectTracker`) uses `HttpProxy`.
